android/src/main/java/com/ibits/react_native_in_app_review/AppReviewModule.java

```diff
@@ -45,16 +45,19 @@ public void show() {
             request.addOnCompleteListener(task -> {
                 if (task.isSuccessful()) {
                     // We can get the ReviewInfo object
-                    ReviewInfo reviewInfo = task.getResult();
-                    Task<Void> flow = manager.launchReviewFlow(getCurrentActivity(), reviewInfo);
-
-                    flow.addOnCompleteListener(taski -> {
-                        // The flow has finished. The API does not indicate whether the user
-                        // reviewed or not, or even whether the review dialog was shown. Thus, no
-                        // matter the result, we continue our app flow.
-                        Log.e("Review isSuccessful", "" + taski.isSuccessful());
-                    });
+                    try {
+                        ReviewInfo reviewInfo = task.getResult();
+                        Task<Void> flow = manager.launchReviewFlow(getCurrentActivity(), reviewInfo);
 
+                        flow.addOnCompleteListener(taski -> {
+                            // The flow has finished. The API does not indicate whether the user
+                            // reviewed or not, or even whether the review dialog was shown. Thus, no
+                            // matter the result, we continue our app flow.
+                            Log.e("Review isSuccessful", "" + taski.isSuccessful());
+                        });
+                    } catch (Exception e) {
+                        Log.e("getResult may have thrown an exception. This is likely an emulated device.");

```

Comment:
You should apply a correct init Log.e(String,String)
it's not building and throws: 
`method Log.e(String,String) is not applicable
      (actual and formal argument lists differ in length)`